Denis_Doubinin/02.Git.Local/readme.md

```diff
@@ -0,0 +1,94 @@
+# Git code
+
+
+   96  git init

```

Comment:
Please use git markdown for reports (for code/log sections too)

Reply:
I have checked and corrected all files readme.md. Now everything is ok, all files are readable from webgui. I'm sorry for this bug.